Sources/ParseLiveQuery/Internal/ClientPrivate.swift

```diff
@@ -209,7 +209,7 @@ extension Client {
             let jsonEncoded = operation.JSONObjectRepresentation
             let jsonData = try NSJSONSerialization.dataWithJSONObject(jsonEncoded, options: NSJSONWritingOptions(rawValue: 0))
             let jsonString = String(data: jsonData, encoding: NSUTF8StringEncoding)
-
+            
```

Comment:
I feel like these whitespace changes are unintentional. Do you think we could revert them?